src/test/java/seedu/duke/food/FoodTest.java

```diff
@@ -1,11 +1,27 @@
 package seedu.duke.food;
 
+import seedu.duke.food.Food;
+
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
 
 class FoodTest {
-    public static void main(String[] args) {
-        Food food = new Food("Kobe Beef", 480,50,40,30);
-        System.out.println(food);
+    private Food testFood;
+
+    @BeforeEach
+    public void setUp(){
+        testFood = new Food("Kobe Beef", 480,50,40,30);
+    }
+
+    @Test
+    public void footTest(){
+        assertEquals(testFood.getCalorie(), 480);
+        assertEquals(testFood.getCarbohydrate(), 50);
+        assertEquals(testFood.getProtein(), 40);
+        assertEquals(testFood.getFats(),30);
```

Comment:
Perhaps you can consider splitting them into individual tests and the consider naming each test in the following three-part format: methodUnderTest_inputGiven_expectedOutput
E.g.
@Test
public void getCalorie_food_returnsCalorie() {
    assertEquals(480, testFood.getCalorie);
}

Also, for the assertEquals method, I think the parameters are (expected, actual) so you might also want to consider switching the variables around.